what3words.dotnet.wrapper/What3WordsV3.cs

```diff
@@ -48,7 +47,7 @@ private void SetupHttpClient(string apiKey, string endpoint, Dictionary<string,
         {
             var httpClient = new HttpClient
             {
-                BaseAddress = new Uri(endpoint.TrimEnd('/') ?? DEFAULT_ENDPOINT)
+                BaseAddress = new Uri(endpoint.TrimEnd('/') + '/' ?? DEFAULT_ENDPOINT)
```

Comment:
This is required for `HttpClient` calls to work. Otherwise, calling `GetAsync` will drop any paths provided on the `BaseAddress`. (i.e. `https://api.what3words.com/v3/autosuggest` will become `https://api.what3words.com/autosuggest`)